torch/csrc/jit/python/pybind_utils.h

```diff
@@ -44,14 +44,31 @@
 #endif
 
 namespace torch {
+
+#ifdef USE_DISTRIBUTED
+namespace distributed {
+namespace rpc {
+class PyRRef;
```

Comment:
Why do we need this here? Existing code uses `torch::distributed::rpc::PyRRef` which does not seem to need this forward declare?

Reply:
Removing.